src/build/app/mod.rs

```diff
@@ -2376,6 +2382,9 @@ impl<'help> App<'help> {
                         $sc.set(AppSettings::GlobalVersion);
                         $sc.version = Some($_self.version.unwrap());
                     }
+                    if $_self.settings.is_set(AppSettings::IgnoreErrors) {
+                        // $sc.set(AppSettings::PartialParsing);
```

Comment:
What is this for?

Reply:
Context: I called `AppSettings::IgnoreErrors` `AppSettings::PartialParsing` in an earlier version of this PR.

The idea here is to propagate the setting. I just wonder if it is really necessary. I couldn't find a test case where it matters. Do you have a suggestion?